Jenkinsfile

```diff
@@ -1,8 +1 @@
-coreJdk11Version="2.161"
-
-buildPlugin(configurations: [
-  [ platform: "linux", jdk: "8", jenkins: null ],
-  [ platform: "windows", jdk: "8", jenkins: null ],
-  [ platform: "linux", jdk: "11", jenkins: coreJdk11Version, javaLevel: "8" ],
-  [ platform: "windows", jdk: "11", jenkins: coreJdk11Version, javaLevel: "8" ]
-])
+buildPlugin(useAci: false, configurations: buildPlugin.recommendedConfigurations())
```

Comment:
`useAci` defaults to `false`, so you could just as easily omit it. But of course this is fine as-is, too.